components/nav.php

```diff
@@ -30,13 +32,13 @@
 				<li class="nav-item">
 					<a class="nav-link" href="/login">Volunteer Login</a>
 				</li>
-			<?php if (isset($_REQUEST['admin'])): ?>
+			<?php if ($USER->hasPermission('can_view_management_tab')): ?>
```

Comment:
Haven't seen the if:endif syntax in awhile but I can't see why not.

Reply:
That was there when I got there. Is there a better way we should do it?